go/cmd/dolt/commands/diff.go

```diff
@@ -403,55 +407,78 @@ func diffSchemas(tableName string, sch1 schema.Schema, sch2 schema.Schema) errha
 		return tags[i] < tags[j]
 	})
 
-	cli.Println("  CREATE TABLE", tableName, "(")
-
-	for _, tag := range tags {
-		dff := diffs[tag]
-		switch dff.DiffType {
-		case diff.SchDiffNone:
-			cli.Println(sql.FmtCol(4, 0, 0, *dff.New))
-		case diff.SchDiffColAdded:
-			cli.Println(color.GreenString("+ " + sql.FmtCol(2, 0, 0, *dff.New)))
-		case diff.SchDiffColRemoved:
-			// removed from sch2
-			cli.Println(color.RedString("- " + sql.FmtCol(2, 0, 0, *dff.Old)))
-		case diff.SchDiffColModified:
-			// changed in sch2
-			oldType, err := dtypes.NomsKindToSqlTypeString(dff.Old.Kind)
-			if err != nil {
-				return errhand.BuildDError("error: failed to diff schemas").AddCause(err).Build()
-			}
-			newType, err := dtypes.NomsKindToSqlTypeString(dff.New.Kind)
-			if err != nil {
-				return errhand.BuildDError("error: failed to diff schemas").AddCause(err).Build()
-			}
-
-			n0, t0 := dff.Old.Name, oldType
-			n1, t1 := dff.New.Name, newType
-
-			nameLen := 0
-			typeLen := 0
-
-			if n0 != n1 {
-				n0 = color.YellowString(n0)
-				n1 = color.YellowString(n1)
-				nameLen = mathutil.Max(len(n0), len(n1))
+	if dArgs.diffOutput == TabularDiffOutput {
+		cli.Println("  CREATE TABLE", tableName, "(")
```

Comment:
Probably worth breaking out functions for each of the two main code paths at this point